en/debug/simulation_debugging.md

```diff
@@ -31,22 +31,15 @@ make px4_sitl_default jmavsim___valgrind
 
 ## Start combinations
 
-SITL can be launched with and without debugger attached and with either jMAVSim or Gazebo Classic as simulation backend.
-This results in the start options below:
+SITL can be launched with and without debugger attached with Gazebo Classic as simulation backend. This results in the start options below:
 
 ```sh
-make px4_sitl_default jmavsim
-make px4_sitl_default jmavsim___gdb
-make px4_sitl_default jmavsim___lldb
-
 make px4_sitl_default gazebo-classic
-make px4_sitl_default gazebo-classic___gdb
-make px4_sitl_default gazebo-classic___lldb
+make px4_sitl_default gazebo-classic_iris_gdb
```

Comment:
@afwilkin Thanks for this. We definitely need to do something, but exactly what depends a bit on what is going to happen in future and what is supported now. 

@beniaminopozzan @Jaeyoung-Lim  Do you know if
- there an intent to bring back the vehicle-less build targets, and 
- whether it is going to happen for both Gazebo Classic and JMAVSim (and Gazebo?).
- there are JMAVSim gdb/lldb targets for individual vehicles? 
- Do all vehicles have them?

@beniaminopozzan I'd just check if `list_vmd_make_targets` if it still existed.

If we can answer some of that, knowing if this is a good approach will be a lot easier. I'm hesitant to delete options without knowing if they are coming back, and at least being able to say what is going to happen in the docs.

Reply:
totally makes sense. I just ran into the issue when trying to do it, so I thought I would add some information. 

What about a note that at the current moment you need to add a vehicle name in order for these gazebo sim to work?

Reply:
when i tried to ripgrep I could not find anything on `list_vmd_make_targets` command

`atlas:~/px4/main/PX4-Autopilot$ rg list_vmd_make_targets
atlas:~/px4/main/PX4-Autopilot$ rg list_vmd
atlas:~/px4/main/PX4-Autopilot$ 
`
But then again I do not 100% know how that command is formed

Reply:
Ah, sorry about that. I was whinging to @beniaminopozzan about the fact that this command USED to exist, and was very very useful. It got removed silently (from my perspective) .

Reply:
> What about a note that at the current moment you need to add a vehicle name in order for these gazebo sim to work?

Yes, that would be simpler/safer for now.

The `list_vmd_make_targets` can also be removed as it isn't planned to come back (yet, though I haven't given up).

Would still be good to know the plan for these other targets, but the note would be enough to help people for now.